src/routes/docs/products/auth/oauth2/+page.markdoc

```diff
@@ -226,7 +249,7 @@ An OAuth 2 [session](/docs/references/cloud/models/session) will have the follow
 
 You can use the `providerAccessToken` to make requests to your OAuth 2 provider. Refer to the docs for the OAuth 2 provider you're using to learn about making API calls with the access token.
 
-# OAuth 2 profile {% #profile %}
+# Refresh tokens {% #refresh-tokens %}
 
 OAuth 2 sessions expire to protect from security risks. OAuth 2 sessions should be refreshed periodically, so access tokens don't expire. Check value of `providerAccessTokenExpiry` to know if the token is expired or is about to expire. Refreshing before every request might cause rate limit problems. You can do this by calling the [Update OAuth Session](/docs/references/cloud/client-web/account#updateSession) endpoint when ever your user visits your app.
```

Comment:
We should clarify the session that expires is the session with the provider and not the Appwrite session.